byte0/Cargo.lock

```diff
@@ -1,237 +0,0 @@
-# This file is automatically @generated by Cargo.
```

Comment:
This wasn't needed now that we are using a workspace!

There is a single `Cargo.lock` at the top level of the repo